src/module/account/settings/_component/FormInput/FormPasswordInput.tsx

```diff
@@ -0,0 +1,56 @@
+import {
+  FormFieldButton,
+  FormGroup,
+  FormGroupField,
+  FormGroupLabel,
+} from '@/shared/components/form/FormElements';
+import { FieldErrors } from 'react-hook-form';
+import FormIcon from './_component/FormIcon';
+import styled from 'styled-components';
+import EyeIcon from 'mdi-react/EyeIcon';
+import Error from '@/shared/components/form/Error';
+import { useState } from 'react';
+
+interface FormInputProps {
+  placeholder: string;
+  errors: FieldErrors;
+  field: { [key: string]: any };
+  isAboveError?: boolean;
+}
+
+export default function FormPasswordInput({
+  placeholder,
+  field,
+  errors,
+  isAboveError,
+}: FormInputProps) {
+  const [passwordShown, setPasswordShown] = useState(false);
+  const { name } = field;
+  return (
+    <FormGroup>
+      <FormGroupLabel>{placeholder}</FormGroupLabel>
+      <FormGroupField>
+        <FormIcon name={name} />
+        <FormInputWrap>
+          <input placeholder={placeholder} type={passwordShown ? 'text' : 'password'} {...field} />
+          {errors[name] && <Error error={errors[name]?.message as string} $top={isAboveError} />}
+        </FormInputWrap>
+        <FormFieldButton
+          $active={passwordShown.toString()}
+          type="button"
+          onClick={() => setPasswordShown((shown) => !shown)}
+        >
+          <EyeIcon />
```

Comment:
btw, a better ux would be to show a open eye icon 🙉 when showing password and a close eye icon 🙈 when not showing password

Reply:
Yes, I totally agree with you! And this improvement is refining in the CP-84
<img width="289" alt="image" src="https://github.com/user-attachments/assets/f065b00c-7545-4354-b0e4-da136692f475">

Reply:
ahhh okay,